deployment/network-operator/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+name: network-operator
+version: 0.2.0-beta
+kubeVersion: '>= 1.15.0'
```

Comment:
why this version? 1.16 we have topology manager so maybe better to change to 1.16?

Reply:
in PRD its 1.15.0